pages/WatchList/index.js

```diff
@@ -11,7 +11,7 @@ export default function WatchList({ movies, bookmarks, onToggleBookmark }) {
     <>
       <StyledH2>Your Watchlist</StyledH2>
```

Comment:
Instead of importing your `MovieCard` and creating a new `MovieList`, you can simply import your `MovieList` component.

```suggestion
 <>
      <StyledH2>Your Watchlist</StyledH2>
      {bookmarkedMovies.length === 0 ? (
        <StyledMessage>No movies here yet...</StyledMessage>
      ) : (
        <MovieList
          movies={bookmarkedMovies}
          onToggleBookmark={onToggleBookmark}
          bookmarks={bookmarks}
        />
      )}
    </>
```